js/frameworks/nextjs.html.markerb

```diff
@@ -141,7 +141,94 @@ our [Vanilla with Candy Sprinkles](https://fly.io/blog/vanilla-candy-sprinkles/)
 blog entry and select the configuration that most closely matches your
 application.  If you still have questions, post on our [community forum](https://community.fly.io/).
 
+### Connecting to Supabase Postgres
 
+[Supabase](https://fly.io/docs/reference/supabase/#create-and-manage-a-supabase-postgres-database) is a managed Postgres service deployed at Fly.io infrastructure.
+
+You can create a Supabase Postgres instance at Fly.io and use the provided `DATABASE_URL`
+as described above. Alternatively, you can connect with a dedicated JavaScript client and leverage its API. In this section we'll help you to get started with such setup.
+
+Start off with provisioning a Supabase project:
+
+<aside class="callout">Running the following command in a Fly.io app context -- inside an app directory or specifying `-a yourapp` -- will automatically pick a region and set secrets on your app.</aside>
+
+```cmd
+flyctl ext supabase create
+```
+```output
+? Choose a name, use the default, or leave blank to generate one: hello-nextjs-db
+? Choose the primary region (can't be changed later) Warsaw, Poland (waw)
+
+Your Supabase database (hello-nextjs-db) in waw is ready. See details and next steps with: https://fly.io/docs/reference/supabase/
+
+Setting the following secrets on hello-nextjs:
+DATABASE_POOLER_URL
+DATABASE_URL
+
+...
+
+-------
+ ✔ [1/2] Machine 1781344b935d38 [app] update succeeded
+ ✔ [2/2] Machine e82de57a07e528 [app] update succeeded
+-------
+```
+
+Next, go to the Supabase Dashboard (`fly ext supabase dashboard`) and copy the _Project URL_ and _API Key_ into an `.env` file in the root of the project:
+
+```env
+SUPABASE_URL=<PROJECT URL>
+SUPABASE_ANON_KEY=<API KEY>
+```
+
+Finally, install the [supabase.js](https://www.npmjs.com/package/@supabase/supabase-js) `npm install @supabase/supabase-js`.
+
+With the above setup in place, let's create a simple page listing TODOs (we'll add some items to the database later) in `app/todos/page.tsx`:
+
+```javascript
+import { createClient } from '@supabase/supabase-js'
+
+const supabaseUrl = process.env.SUPABASE_URL ?? ''
+const supabaseKey = process.env.SUPABASE_ANON_KEY ?? ''
+
+export default async function Page() {
+    const supabase = createClient(supabaseUrl, supabaseKey)
+    const { data: todos } = await supabase.from('todos').select()
+
+    return (
+        <main className="flex min-h-screen flex-col items-center justify-between p-24">
+                <ul>
+                    {todos?.map((todo) => (
+                        <li key={todo.id} className="flex w-full justify-center border-b border-gray-300 bg-gradient-to-b from-zinc-200 pb-6 pt-8 backdrop-blur-2xl dark:border-neutral-800 dark:bg-zinc-800/30 dark:from-inherit lg:static lg:w-auto  lg:rounded-xl lg:border lg:bg-gray-200 lg:p-4 lg:dark:bg-zinc-800/30">
+                            {todo.title}
+                        </li>
+                    ))}
+                </ul>
+        </main>
+    )
+}
```

Comment:
Use 2 spaces for tabs instead of 4.

Reply:
Also, I would remove the Tailwind classes, as its not really relevant to this demo and not everyone uses it.

Reply:
<img width="643" alt="image" src="https://github.com/superfly/docs/assets/748608/3c71130a-ef6d-4a4f-827f-58b64d5a8714">

Reply:
lovely!!!